spread.yaml

```diff
@@ -126,6 +126,9 @@ backends:
             - ubuntu-20.04-64:
                   storage: 12G
                   workers: 8
+            - ubuntu-24.10-64:
+                  storage: 12G
+                  workers: 8
```

Comment:
a nitpick for master, not here: but shouldn't these backends be ordered somehow?